lib/adagrams.rb

```diff
@@ -0,0 +1,157 @@
+def draw_letters
+  letter_pool = []
+  9.times do
+    letter_pool << "A"
+    letter_pool << "I"
+  end
+
+  2.times do
+    letter_pool << "B"
+    letter_pool << "C"
+    letter_pool << "F"
+    letter_pool << "H"
+    letter_pool << "M"
+    letter_pool << "P"
+    letter_pool << "V"
+    letter_pool << "W"
+    letter_pool << "Y"
+  end
+
+  4.times do
+    letter_pool << "D"
+    letter_pool << "L"
+    letter_pool << "S"
+    letter_pool << "U"
+  end
+
+  12.times do
+    letter_pool << "E"
+  end
+
+  3.times do
+    letter_pool << "G"
+  end
+  1.times do
+    letter_pool << "J"
+    letter_pool << "K"
+    letter_pool << "Q"
+    letter_pool << "X"
+    letter_pool << "Z"
+  end
+
+  6.times do
+    letter_pool << "N"
+    letter_pool << "R"
+    letter_pool << "T"
+  end
+
+  8.times do
+    letter_pool << "O"
+  end
+
+  letter_pool.sort!
+
+  rand_letter = []
+  10.times do
+    rand_letter << letter_pool.sample
+  end
+  return rand_letter
+end
+
+letters_in_hand = draw_letters
```

Comment:
Since this is library code, you'll want to delete this line, since it doesn't get used or run anywhere (or at least, doesn't get used by the tests!)